src/Umbraco.Web.Common/RuntimeMinification/SmidgeRuntimeMinifier.cs

```diff
@@ -85,7 +85,8 @@ public void CreateJsBundle(string bundleName, params string[] filePaths)
             // Here we could configure bundle options instead of using smidge's global defaults.
             // For example we can use our own custom cache buster for this bundle without having the global one
             // affect this or vice versa.
-            var bundle = _bundles.Create(bundleName, _jsPipeline.Value, WebFileType.Js, filePaths);
+            var pipeline = optimizeOutput ? _jsPipeline.Value : _bundles.PipelineFactory.Create();
```

Comment:
Just realized this switch statement will use the default pipeline which we don't want to do, for non-optimized output we should use our own pipeline since we don't want to rely on globals. I'll update.